pkg/executor/infoschema_reader.go

```diff
@@ -3691,7 +3691,7 @@ func (e *memtableRetriever) setDataForClusterIndexUsage(ctx context.Context, sct
 	dom := domain.GetDomain(sctx)
 	rows := make([][]types.Datum, 0, 100)
 	checker := privilege.GetPrivilegeManager(sctx)
-	extractor, ok := e.extractor.(*plannercore.InfoSchemaBaseExtractor)
+	extractor, ok := e.extractor.(*plannercore.InfoSchemaIndexUsageExtractor)
```

Comment:
Maybe add a method `ListSchemasAndTables` for `InfoSchemaIndexUsageExtractor` and avoid using `SchemaTableInfos`.

Reply:
`InfoSchemaIndexUsageExtractor` has already implemented schema and table filter, which is used in tidb_index_usage.
The only difference is that for cluster_tidb_index_usage, the request needs to be sent to all tidb instances for execution